src/plugins/offline/azureOfflinePlugin.ts

```diff
@@ -30,7 +30,13 @@ export class AzureOfflinePlugin extends AzureBasePlugin {
             usage: "Start Azure Function app - assumes offline build has already occurred",
             lifecycleEvents: [
               "start"
-            ]
+            ],
```

Comment:
I'm assuming default behavior is to cleanup, correct?

Reply:
Yes